rust-runtime/aws-smithy-client/Cargo.toml

```diff
@@ -9,11 +9,11 @@ repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
 rt-tokio = ["aws-smithy-async/rt-tokio"]
-test-util = ["aws-smithy-protocol-test", "serde/derive", "rustls"]
-native-tls = ["client-hyper", "hyper-tls", "rt-tokio"]
-rustls = ["client-hyper", "hyper-rustls", "rt-tokio", "lazy_static"]
-client-hyper = ["hyper"]
-hyper-webpki-doctest-only = ["hyper-rustls/webpki-roots"]
+test-util = ["dep:aws-smithy-protocol-test", "dep:serde", "serde?/derive", "rustls"]
```

Comment:
Those are a good so that if you move some of them to another feature. I will remove them anyway.

Reply:
actually had to put it back because it fails with this error if i don't:
> the package `dynamo-tests` depends on `aws-smithy-client`, with features: `serde` but `aws-smithy-client` does not have these features.
 It has an optional dependency with that name, but but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.